content/chainguard/chainguard-images/shared-responsibility-model/index.md

```diff
@@ -0,0 +1,167 @@
+
+The next few sections dig deeper into the details about App Images, Base Images, and the responsibilities of Chainguard and its customers for both.
+
+
+## App Images
+Chainguard App Images are specialized container images designed to be used "as-is" in production environments.
+
+### Key Characteristics
+1. Minimal Attack Surface: Built using a [distroless](/chainguard/chainguard-images/getting-started-distroless/) approach, removing unnecessary components to reduce potential vulnerabilities.
+2. Up-to-date Components: Continuously updated with the latest security patches and version updates from upstream projects.
+3. Low to Zero CVEs: Rigorously scanned and maintained to minimize known vulnerabilities.
+4. Layered Security Responsibility:
+    * Top layers (Apps and Lang Libs) are managed by upstream open source projects
+    * Lower layers (Toolchains and System) are secured and maintained by Chainguard
+5. Compliance with CHP Levels: Adhere to Chainguard's Container Hardening Policy, typically at higher levels (CHP3-CHP5) for enhanced security.
+6. SLSA Conformance: Built in accordance with [Supply-chain Levels for Software Artifacts](/open-source/slsa/what-is-slsa/) (SLSA) guidelines.
+7. Immutable and Verified: Signed and verified to ensure integrity throughout the software supply chain.
+8. SBOM Generation: Includes a build-time generated [Software Bill of Materials](/open-source/sbom/what-is-an-sbom/) for transparency and auditing.
+9. FIPS Compliance: Versions available that meet Federal Information Processing Standards.
+10. Rapid Patching: Covered by Chainguard's SLA for quick resolution of newly discovered vulnerabilities.
+
+### Use Cases
+* Ideal for organizations requiring secure, production-ready container images
+* Suitable for environments with stringent security and compliance requirements
+* Designed to integrate seamlessly into modern CI/CD pipelines and cloud-native architectures
+
+### App Images Shared Responsibilities
+* Apps Layer: Management of CVEs in the top-level application code.
+* Lang Libs Layer: Handling vulnerabilities in language-specific libraries built into the image (e.g., `go mod`, `pom`).
+
+Should a CVE be detected in any of these images we would likely bump a dependency to avoid in the next build. The outcome is two-fold: either bumping the dependency is successful or it breaks break the image. 
```

Comment:
First sentence needs rewritten. Maybe:

"If a CVE is detected in one of these layers and is fixable by updating a dependency, we will attempt to do so."?

In general I don't like the term "bumping" for customer facing docs, but that could just be me :)

Reply:
I guess the second sentence should be: "There are two possible outcomes; ..."

To me "two-fold" implies both things are true, not a binary decision.